html/changelogs/air_pump_weight_class_bugfix.yml

```diff
@@ -0,0 +1,6 @@
+author: SleepyGem
+
+delete-after: True
+
+changes:
+  - bugfix: "Changed the portable air pump- and scrubber weight classes to one representing their size more accurately."
```

Comment:
I think it is important to actually describe what effect this will have on the player, in the change log. Say it won't fit in bags anymore, they don't know what weight classes do.

Reply:
fixed.